transports/transport-rest/transport-rest-vertx/src/main/java/io/servicecomb/transport/rest/vertx/accesslog/element/impl/BytesWrittenV2Element.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.transport.rest.vertx.accesslog.element.impl;
+
+import io.servicecomb.transport.rest.vertx.accesslog.AccessLogParam;
+import io.servicecomb.transport.rest.vertx.accesslog.element.AccessLogElement;
+import io.vertx.core.http.HttpServerResponse;
+
+/**
+ * Zero bytes written as -
+ */
+public class BytesWrittenV2Element implements AccessLogElement {
+
+  public static final String ZERO_BYTES = "-";
+
+  @Override
+  public String getFormattedElement(AccessLogParam accessLogParam) {
+    HttpServerResponse response = accessLogParam.getRoutingContext().response();
+    if (null == response) {
+      return ZERO_BYTES;
+    }
+
+    long bytesWritten = response.bytesWritten();
+    if (0 == bytesWritten) {
+      return ZERO_BYTES;
+    } else {
```

Comment:
already return, no need else

Reply:
If bytesWritten equals to 0, "-" will be returned instead of "0". So there should be this check logic.

Reply:
check is no problem
"else" is unnecessary